src/System.IO.IsolatedStorage/src/ApiCompatBaseline.txt

```diff
@@ -1,3 +1,4 @@
 Compat issues with assembly System.IO.IsolatedStorage:
 CannotAddAbstractMembers : Member 'System.IO.IsolatedStorage.IsolatedStorage.GetPermission(System.Security.PermissionSet)' is abstract in the implementation but is missing in the contract.
-Total Issues: 1
+MembersMustExist : Member 'System.IO.IsolatedStorage.IsolatedStorageException..ctor(System.Runtime.Serialization.SerializationInfo, System.Runtime.Serialization.StreamingContext)' does not exist in the implementation but it does exist in the contract.
```

Comment:
I assume this is only for one of the build configurations? If so we should put in a configuration specific ApiCompatBaseline.netcore50.txt (or whichever config it is).